src/common/traffic-source.js

```diff
@@ -4,14 +4,13 @@ import { CookieStorage, LocalStore } from './storage'
 const TrafficSource = (() => {
     let cookie
 
-    const initCookie = () => {
-        if (!cookie) {
-            cookie = new CookieStorage('utm_data')
-            cookie.read()
-            // expiration date is used when writing cookie
-            const now = new Date()
-            cookie.expires = now.setMonth(now.getMonth() + 3)
-        }
+    const initUtmCookie = () => {
+        cookie = new CookieStorage('utm_data')
+        cookie.remove()
```

Comment:
we need to remove here even though it's declared as a `new CookieStorage` on the line before?

Reply:
to select the `utm_data` cookie by `storage` service, we need to do `new CookieStorage('utm_data')`, then we can remove the previous data. but in this case, we dont need `cookie.read()` anymore